src/gt4py/next/program_processors/runners/dace_fieldview/gtir_dataflow.py

```diff
@@ -863,7 +998,27 @@ def _visit_generic_builtin(self, node: gtir.FunCall) -> DataExpr:
                     connector,
                 )
 
-        return self._construct_tasklet_result(dtype, tasklet_node, "result")
+        if isinstance(node.type, itir_ts.ListType):
+            # The only builtin function (so far) handled here that returns a list
+            # is 'make_const_list'. There are other builtin functions (map_, neighbors)
+            # that return a list but they are handled in specialized visit methods.
+            # This method (the generic visitor for builtin functions) always returns
+            # scalars. This is also the case of 'make_const_list' expression: it simply
```

Comment:
I think it is not super precise here.
The function does not _always_ return a scalar, in case of the `make_const_list()` the function returns an array of shape `(1,)` which is close to an array but not the same (even in DaCe you can use a `(1,)` arry only in ~90% of the cases were an array is needed (but this is probably a shortcoming in the implementation).
So you should make that a bit more precise.

Reply:
Done.